cmd/internal/find/main.go

```diff
@@ -67,15 +67,13 @@ func Main(cmd *cobra.Command, args []string) int {
 	// Do the walk
 	conn := client.ForUNIXSocket(config.Socket)
 	walker := newWalker(result, conn)
+	exitCode := 0
 	for _, path := range result.Paths {
-		e, err := info(conn, path)
-		if err != nil {
-			cmdutil.ErrPrintf("%v\n", err)
-			continue
+		if !walker.Walk(path) {
+			exitCode = 1
```

Comment:
I'm wondering if this might be a bit too strict, because unlike BSD/GNU find, `wash find` makes API requests across a network (which increases the likelihood of errors). Maybe it's best to make it consistent for now, and if users find it annoying, we can revisit the exit code stuff later. I vote for the latter option.

Reply:
Yeah, I'm not sure I'll like it. I definitely get access denied mixed in with my searches. It's currently easy to filter that out with `>/dev/null`. But it's also reasonable to error when that happens and I can `|| true`. So 🤷‍♂ .